modules/dynamic/src/smithy4s/dynamic/DynamicEndpoint.scala

```diff
@@ -0,0 +1,21 @@
+package smithy4s
+package dynamic
+
+case class DynamicEndpoint(
+    namespace: String,
+    name: String,
+    input: Schema[DynData],
```

Comment:
need to add the `Input` / `Output` hint there.

Reply:
Added in https://github.com/disneystreaming/smithy4s/pull/17/commits/41e5d1fea6e7fc825a397828578c0cc727d8ca28